code/WorkInProgress/JobXPRewards.dm

```diff
@@ -522,6 +522,27 @@ mob/verb/checkrewards()
 
 /////////////Bartender////////////////
 
+/datum/jobXpReward/bartender/spectromonocle
+	name = "Spectroscopic Monocle"
+	desc = "Now you can look dapper and know which drinks you poisoned at the same time"
+	required_levels = list("Bartender"=10)
+	icon_state = "?"
+	claimable = 1
+	var/path_to_spawn = /obj/item/clothing/glasses/spectro/monocle
+
+	activate(var/client/C)
+		var/obj/item/clothing/glasses/spectro/glasses = locate(/obj/item/clothing/glasses/spectro) in C.mob.contents
```

Comment:
You may wanna do `var/glasses = C.mob.find_in_hand(/obj/item/clothing/glasses/spectro)`.

Reply:
done